test/zdtm.py

```diff
@@ -202,6 +202,8 @@ def __copy_one(self, fname):
 
     def __copy_libs(self, binary):
         ldd = subprocess.Popen(["ldd", binary], stdout=subprocess.PIPE)
+        stdout, stderr = ldd.communicate()
```

Comment:
`stderr` is not used, we could use `unused` or `_` instead
https://www.python.org/dev/peps/pep-0640/

Reply:
Fixed